### ISSUE TITLE
Installation on Ubuntu

### ISSUE THREAD

**Issue by pitfisher** [Enchufa2/video-tester#2]
Is there anyone succeded in installing this software on Ubuntu? I'm trying to do this right now, but it seems that some packages either not available anymore or never had been presented in Ubuntu repositories.

Best regards.

**Comment by Enchufa2** [comment]
The issue is that the same packages have different names in Ubuntu compared to Fedora. Using the package list in the README, you'll need to do some search by keyword to figure out which are the equivalent ones. For example, it seems that gstreamer v1 packages are prefixed "gstreamer1.0" instead of "gstreamer1" in Ubuntu according to [this search](https://packages.ubuntu.com/search?suite=all&arch=any&searchon=names&keywords=gstreamer).

I don't use Ubuntu, so it would be a nice addition to the README if you could find all these package names.

**Comment by pitfisher** [comment]
Here it is: command to get all dependencies using apt package manager:
`sudo apt install libgstreamer1.0-0 gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly gstreamer1.0-libav gstreamer1.0-doc gstreamer1.0-tools gstreamer1.0-x gstreamer1.0-alsa gstreamer1.0-gl gstreamer1.0-gtk3 gstreamer1.0-qt5 gstreamer1.0-pulseaudio gstreamer1.0-rtsp python-gobject python-libpcap python-numpy python-matplotlib  python-wxmpl python-wxgtk3.0 python-opencv gir1.2-gst-rtsp-server-1.0 libgstrtspserver-1.0-0 
`
Only thing I had to fix manually before I could launch your app is that I changed line where OpenCV is imported in vq.py - I altered import math, cv to import math, cv2. It doesn't fix compatibility with OpenCV 3 but at least allows to launch the app.

I will make a pull request tomorrow.
